stdlib/buffer.gr

```diff
@@ -96,674 +96,691 @@ let checkIsIndexInBounds = (i, len, buf) => {
   if (i >= buf.len || i + len > buf.len) throw Exception.IndexOutOfBounds
 }
 
+let addInt8help = (value, buffer) => {
+  autogrow(_8BIT_LEN, buffer)
+  let index = buffer.len
+  buffer.len = buffer.len + _8BIT_LEN
+  Bytes.setInt8(index, value, buffer.data)
+}
+
+let addInt16help = (value, buffer) => {
+  autogrow(_16BIT_LEN, buffer)
+  let index = buffer.len
+  buffer.len = buffer.len + _16BIT_LEN
+  Bytes.setInt16(index, value, buffer.data)
+}
+
+let addInt32help = (value, buffer) => {
+  autogrow(_32BIT_LEN, buffer)
+  let index = buffer.len
+  buffer.len = buffer.len + _32BIT_LEN
+  Bytes.setInt32(index, value, buffer.data)
+}
```

Comment:
I added these help functions which are needed by `addChar` - in addition, they no longer call the `set*` functions that also do a index assertion. It's probably a tiny performance improvement, but I don't think they need to do the assertion since they grow the buffer - right @jozanza?